### PR TITLE
Registration form

### PR DESCRIPTION
**Pull request by dg1886**

### CODE REVIEW

File: src/Components/Button/styled.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+
+export const WrapperButton = styled.button`
+  display: inline-block;
+  width: 276px;
+  height: 63px;
+  background-color: ${p => p.backgroundColor};
```
Comment (Nehved) [inline src/Components/Button/styled.js]: color: ${({ color }) => color};

File: src/Components/Button/styled.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  margin: 1em;
+  padding: 0.25em 1em;
+  border-radius: 8px;
+  cursor: pointer;
+  border: none;
+  color: ${p => p.color};
```
Comment (Nehved) [inline src/Components/Button/styled.js]: the same

File: src/Components/Button/index.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Button.propTypes = {
+    color: string,
+    backgroundColor: string,
+    text: string,
+}
+
+Button.defaultProps = {
+    color: '#f4f9f9',
+    backgroundColor: '#006162',
+    text: 'Submit'
```
Comment (Nehved) [inline src/Components/Button/index.jsx]: width: 100%

File: src/Components/Button/styled.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+
+export const WrapperButton = styled.button`
+  display: inline-block;
+  width: 276px;
```
Comment (Nehved) [inline src/Components/Button/styled.js]: pass through props

File: src/Components/Button/styled.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  height: 63px;
```
Comment (Nehved) [inline src/Components/Button/styled.js]: the same

File: src/Components/Inputs/styled.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const WrapperInput = styled.input`
+  width: 400px;
```
Comment (Nehved) [inline src/Components/Inputs/styled.js]: the same

File: src/Components/Inputs/styled.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  border-radius: 8px;
+  background-color: #FFFFFF;
+  border: 1px solid #A2CDCD;
+  color: #778686;
+  font-size: 1.4em;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: 0.05em;
+  padding-left: 10px;
+  type=${p => p.type};
```
Comment (Nehved) [inline src/Components/Inputs/styled.js]: remove

File: src/Components/Inputs/styled.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  placeholder=${p => p.placeholder};
```
Comment (Nehved) [inline src/Components/Inputs/styled.js]: remove

File: src/Components/RegistrationForm/Form.jsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import {FormAreaWrapper, FormPageWrapper} from "./styled";
+import Button from "../Button";
+import {Input} from "../Inputs/Input";
+import {Title} from "../Title/Title";
```
Comment (Nehved) [inline src/Components/RegistrationForm/Form.jsx]: export default

File: src/Components/Button/styled.js (hunk 1 above)
Comment (dg1886) [reply src/Components/Button/styled.js]: fixed

File: src/Components/Button/styled.js (hunk 2 above)
Comment (dg1886) [reply src/Components/Button/styled.js]: fixed

File: src/Components/Button/index.jsx (hunk 3 above)
Comment (dg1886) [reply src/Components/Button/index.jsx]: fixed

File: src/Components/RegistrationForm/Form.jsx (hunk 9 above)
Comment (dg1886) [reply src/Components/RegistrationForm/Form.jsx]: fixed

File: src/Components/Inputs/styled.js (hunk 8 above)
Comment (dg1886) [reply src/Components/Inputs/styled.js]: fixed

File: src/Components/Inputs/styled.js (hunk 7 above)
Comment (dg1886) [reply src/Components/Inputs/styled.js]: fixed

File: src/Components/Button/styled.js (hunk 4 above)
Comment (dg1886) [reply src/Components/Button/styled.js]: fixed

File: src/Components/Button/styled.js (hunk 5 above)
Comment (dg1886) [reply src/Components/Button/styled.js]: fixed

File: src/Components/Inputs/styled.js (hunk 6 above)
Comment (dg1886) [reply src/Components/Inputs/styled.js]: fixed

File: src/App.jsx [hunk 10]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import GlobalFonts from './Fonts/Fonts';
+
+import RegistrationForm from './Components/RegistrationForm/Form';
+
+function App() {
+  return (
+    <div className="App">
+      <GlobalFonts />
```
Comment (Nehved) [inline src/App.jsx]: remove

File: src/Components/Button/index.jsx [hunk 11]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
```
Comment (Nehved) [inline src/Components/Button/index.jsx:2]: remove empty line

File: src/Components/Inputs/Input.jsx [hunk 12]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { string } from 'prop-types';
+import { WrapperInput, Wrapper } from './styled';
+
+const Input = ({
+  type, placeholder, width, height,
+}) => (
+  <Wrapper width={width} height={height}>
```
Comment (Nehved) [inline src/Components/Inputs/Input.jsx]: remove redundant wrapper, pass width and height straight to input

File: src/Components/Inputs/styled.js [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+  border: 1px solid #A2CDCD;
+  color: #778686;
+  font-size: 1.4em;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: 0.05em;
+  padding-left: 10px;
+`;
+
+export const Wrapper = styled.div`
```
Comment (Nehved) [inline src/Components/Inputs/styled.js]: remove

File: src/Components/RegistrationForm/Form.jsx [hunk 14]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PageWrapper, FormWrapper } from './styled';
+import Button from '../Button';
+import Input from '../Inputs/Input';
+import Title from '../Title/Title';
+
+const RegistrationForm = () => (
+  <PageWrapper>
+    <FormWrapper>
+      <Title fontSize="2em" />
```
Comment (Nehved) [inline src/Components/RegistrationForm/Form.jsx]: convert to px

File: src/Components/Title/styled.js [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+
+export const TitleWrapper = styled.h3`
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 700;
+  display: inline-block;
+  letter-spacing: 0.05em;
+  color: ${(p) => p.color};
+  font-size: ${(p) => p.fontSize};
+
```
Comment (Nehved) [inline src/Components/Title/styled.js]: empty line

File: src/Fonts/Fonts.js [hunk 16]
```diff
@@ -0,0 +1,40 @@
+import { createGlobalStyle } from 'styled-components';
```
Comment (Nehved) [inline src/Fonts/Fonts.js]: remove component

File: src/index.js [hunk 17]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import './index.css';
```
Comment (Nehved) [inline src/index.js]: uncomment

File: src/Components/RegistrationForm/Form.jsx (hunk 14 above)
Comment (dg1886) [reply src/Components/RegistrationForm/Form.jsx]: fixed

File: src/Components/Inputs/styled.js (hunk 13 above)
Comment (dg1886) [reply src/Components/Inputs/styled.js]: fixed

File: src/Components/Inputs/Input.jsx (hunk 12 above)
Comment (dg1886) [reply src/Components/Inputs/Input.jsx]: fixed and create InputWrapper with height and width;

**Review by Nehved — APPROVED** [review]

**Pull request merged**